packages/integration-tests/cmr.js

```diff
@@ -118,24 +119,15 @@ const sampleUmmGranule = {
  * Returns true if the concept exists - if the cmrLink
  * returns a 200 and there are entries
  *
- * @param {string} cmrLink - CMR URL path to concept,
- * i.e. what is returned from post to cmr task
+ * @param {string} cmrLink
+ *   CMR URL path to concept, i.e. what is returned from post to cmr task
+ *   See `@cumulus/cmrjs/cmr-utils/publishECHO10XML2CMR` for expected URL
  * @returns {boolean} true if the concept exists in CMR, false if not
  */
 async function conceptExists(cmrLink) {
-  let response;
-  try {
-    response = await got.get(cmrLink, { json: true });
-  }
-  catch (err) {
-    log.error(err);
```

Comment:
👍   I'm unclear on why we were doing this

Reply:
It was some additional logging for debugging that I forgot to cleanup previously